app/controllers/canonical_transactions_controller.rb

```diff
@@ -3,6 +3,7 @@
 require "csv"
 
 class CanonicalTransactionsController < ApplicationController
+  include TurboStreamFlash
```

Comment:
Thoughts on including this in `ApplicationController`?

Reply:
> Thoughts on including this in `ApplicationController`?

That was my original plan but then I realized `ApplicationController` is also used for the API where this functionality is useless 🤷. For now I think this is alright and we can reconsider if we see broader usage.